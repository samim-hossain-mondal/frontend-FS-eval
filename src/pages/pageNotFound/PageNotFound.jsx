import React from 'react';
import './PageNotFound.css';

export default function PageNotFound() {
  return (
    <div className="page-not-found-text">
      <h3>Page not found!</h3>
    </div>
  );
}