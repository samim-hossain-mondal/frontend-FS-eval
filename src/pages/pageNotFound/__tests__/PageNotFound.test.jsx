import React from 'react';
import { render } from '@testing-library/react';
import PageNotFound from '../PageNotFound';

describe('PageNotFound', () => {
  it('should render successfully', () => {
    const { container } = render(<PageNotFound />);
    expect(container).toMatchSnapshot();
  });
});