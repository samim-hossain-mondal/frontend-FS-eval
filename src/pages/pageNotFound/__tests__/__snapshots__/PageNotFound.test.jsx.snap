// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound should render successfully 1`] = `
<div>
  <div
    class="page-not-found-text"
  >
    <h3>
      Page not found!
    </h3>
  </div>
</div>
`;
